client/src/components/App.jsx

```diff
@@ -4,8 +4,10 @@ import Search from './Search.jsx';
 import Ratings from './Ratings.jsx';
 import ReviewsList from './ReviewsList.jsx';
 import MainRating from './MainRating.jsx';
+import SearchDescription from './SearchDescription.jsx';
 import styles from './App.css';
```

Comment:
How is your resolve set in webpack? You can make it so you do not require .jsx/.css etc.